### PR TITLE
R1face mic

### PR DESCRIPTION
**Pull request by barbalberto**
This PR add the yarp device handling the microphones in the R1 head.

**NOTE:**
This PR depends on the yarp PR https://github.com/robotology/yarp/pull/1560 which fixes a bug in the yarp::sig::Sound class

### CODE REVIEW

**Comment by pattacini** [comment]
Hi @barbalberto you have to open the PR against `devel`, I suppose

**Comment by barbalberto** [comment]
Ops, right! Fixed

Thank @pattacini

**Comment by barbalberto** [comment]
Driver fixed and succesfully tested.
PR ready to be merged

File: cermod/r1face_mic/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright: (C) 2015 iCub Facility
```
Comment (vtikha) [inline cermod/r1face_mic/CMakeLists.txt]: might be better to have a recent date ;-)

**Review by vtikha — APPROVED** [review]
Looks good to me

File: cermod/r1face_mic/CMakeLists.txt (hunk 1 above)
Comment (barbalberto) [reply cermod/r1face_mic/CMakeLists.txt]: Done, thanks

**Pull request merged**